### DISCUSSION TITLE
Capacity and reported usage

### DISCUSSION THREAD

**Discussion by AlexZIX** [minio/minio#20650] [Q&A] [answered]
I have my MinIO integrated with Prometheus. I see two metrics: reported usage and used capacity. If I understand right the reported usage is total of real sizes of uncompressed objects and used capacity is the really used space on disks. Can somebody confirm that it's right? I just have 27 TB reported usage and 19 TB used capacity for now and this this difference because of compression enabled.

**Comment by klauspost** [comment, marked as answer]
Correct.

**Reply by AlexZIX** [reply]
Thanks
